apis/kueue/webhooks/workload_webhook.go

```diff
@@ -63,26 +74,84 @@ func (w *WorkloadWebhook) Default(ctx context.Context, obj runtime.Object) error
 	}
 	for i := range wl.Spec.PodSets {
 		podSet := &wl.Spec.PodSets[i]
-		setContainersDefaults(podSet.Template.Spec.InitContainers)
-		setContainersDefaults(podSet.Template.Spec.Containers)
+
+		// we apply the default request logic in a pod spec copy in order to compute the TotalResources
+		specCopy := podSet.Template.Spec.DeepCopy()
+		updatePodDefaultResources(specCopy, nsLimitRangesDefaults)
```

Comment:
let's apply these defaults into the totals only, instead of modifying the pod template